homeassistant/components/swiss_public_transport/strings.json

```diff
@@ -49,12 +49,37 @@
       }
     }
   },
+  "services": {
+    "fetch_connections": {
+      "name": "Fetch Connections",
+      "description": "Fetch a list of connections from the swiss public transport.",
```

Comment:
As discussed on Discord:
Either the name of the integration should be translated, or we should inject the untranslated domain name here.